src/jpack.js

```diff
@@ -41,8 +41,16 @@
 
         switch (schema.type) {
         case 'array':
-            schema.items = {}
-            schemaIter(node[0], schema.items)
```

Comment:
No, you don't understand what json schema is. An array can only have one type of items. Something like `[1, 'str']` is illegal.